src/components/PrivacyModal/PrivacyModal.test.tsx

```diff
@@ -83,4 +85,91 @@ describe('Test PrivacyModal', () => {
     )
     expect(form.queryByText('After reading please accept:')).not.toBeInTheDocument()
   })
+
+  test('checkUniversity returns empty', async() => {
+    mockServices.getUser.mockImplementationOnce(() =>
+      Promise.resolve({
+        id: 1,
+        lms_user_id: 1,
+        name: 'Thaddäus Tentakel',
+        role: 'Tester',
+        role_id: 1,
+        settings: {
+          id: 1,
+          user_id: 1,
+          pswd: '1234',
+          theme: 'test'
+        },
+        university: ''
+      })
+    )
+    const { fetchUser } = usePersistedStore.getState()
+    const result = await fetchUser()
+    const form = render(
+      <MemoryRouter initialEntries={['/projectinformation']}>
+        <PrivacyModal />
+      </MemoryRouter>
+    )
+    const declineButton = form.getByRole('button', { name: /components.PrivacyModal.returnToMoodle/i })
+    fireEvent.click(declineButton)
+  })
```

Comment:
Zusätzlich noch die Imports für renderHook und usePrivacyModal
```suggestion
  test('fetchUser fails', async () => {
    mockServices.getUser = jest.fn().mockImplementationOnce(() => Promise.reject(new Error('error')))

    const { result } = renderHook(() => usePrivacyModal(), {
      wrapper: ({ children }) => <MemoryRouter initialEntries={['/projectinformation']}>{children}</MemoryRouter>

    })

    expect(await result.current.checkUniversity()).toBe('')
  })
```